packages/itwinui-css/src/surface/surface.scss

```diff
@@ -2,19 +2,19 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 
 @mixin iui-surface(
-  $background-color: var(--iui-color-background-1),
+  $background-color: var(--iui-color-background),
+  $border-color: var(--iui-color-border-subtle),
   $border-radius: 0,
   $elevation: var(--iui-shadow-1)
 ) {
   --iui-surface-background-color: #{$background-color};
+  --iui-surface-border-color: #{$border-color};
   --iui-surface-border-radius: #{$border-radius};
   --iui-surface-elevation: #{$elevation};
 
   background-color: var(--iui-surface-background-color);
+  border: 1px solid var(--iui-surface-border-color);
   border-radius: var(--iui-surface-border-radius);
   box-shadow: var(--iui-surface-elevation);
-  color: var(--iui-color-foreground-2);
-  @media (forced-colors: active) {
-    border: 1px solid transparent;
-  }
+  color: var(--iui-color-text);
```

Comment:
@mayank99, should `color` be removed from `surface.scss` for the same reason mentioned here? https://github.com/iTwin/iTwinUI/pull/762/#discussion_r1013071131

Reply:
i think it's fine to keep color in surface because it's intended to be a starting point for our users